README.md

```diff
@@ -47,7 +47,10 @@ which ssm-sh
 $ ssm-sh --help
 
 Usage:
-  ssm-sh [OPTIONS] <list | run | shell>
+  main [OPTIONS] <command>
```

Comment:
Would you mind changing this to `ssm-sh` rather than `main`?
I remember that I had to update it manually when I generated the doc last time..(Not sure why yet. We should figure out why this is printing `main` so that we no longer need to care 😉

Reply:
Good catch, will fix :)